jaeger-core/src/main/java/io/jaegertracing/Configuration.java

```diff
@@ -653,19 +652,19 @@ public static SenderConfiguration fromEnv() {
     }
   }
 
-  private static String stringOrDefault(String value, String defaultValue) {
+  protected static String stringOrDefault(String value, String defaultValue) {
```

Comment:
Are you making them `protected` so that you can *use* them in subclasses? Those are not part of the business of this class, I'd rather see them in a, say, `StringUtils` class (as much as a I dislike `utils` classes).

Reply:
Honestly, my subclass doesn't use these. I thought if they were `static private` utility methods here, then they ought to be `static protected` for subclasses. Nonetheless, I was considering adding a new class called `ConfigUtils` or something just for that purpose.